components/resources/jobExplorer/JobExplorerJobComponent.tsx

```diff
@@ -74,7 +73,7 @@ const JobExplorerJobComponent = ({
         <div className="mb-8">
           <h2 className="mb-4 text-2xl font-bold text-blue-600">
             <a
-              href={project.link}
+              href={'projects/' + project.link}
```

Comment:
Added this 'projects/' link to appropriately point towards `projects/[slug].tsx`